src/content_cao.cpp

```diff
@@ -1501,7 +1501,14 @@ void GenericCAO::processMessage(const std::string &data)
 	if (cmd == GENERIC_CMD_SET_PROPERTIES) {
 		m_prop = gob_read_set_properties(is);
 
-		m_selection_box = m_prop.collisionbox;
+		aabb3f no_selection = aabb3f(0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f);
```

Comment:
This is an ugly workaround. Rather extend `ClientEnvironment::getSelectedActiveObject` to skip the objects that aren't pointable.